extra_geom/pyfai.py

```diff
@@ -42,3 +42,13 @@ class LPD1M(Detector):
 
     def __init__(self, pixel1=5e-4, pixel2=5e-4, **kwargs):
         super().__init__(pixel1, pixel2, **kwargs)
+
+
+class JUNGFRAU(Detector):
```

Comment:
From the discussion with Jérôme, we should call this something like JUNGFRAU_EuXFEFL to ensure we don't clash with the names already in PyFAI's class registry (and remove the alias).

Reply:
Thanks, renamed in https://github.com/European-XFEL/EXtra-geom/pull/197/commits/77df7fc01730ecb3fc36e78e624f64b43f3e9dcd.